2020-01-Winter/5_graphs/README.md

```diff
@@ -138,21 +164,45 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s3"/>
 
-### 3. SOLUTION 3 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution 
-
-TODO :bug:
-
-#### Optimal Solution
-
-TODO :bug:
+### 3. Is Graph Bipartite
+
+Source: [Leetcode](https://leetcode.com/problems/is-graph-bipartite/)
+
+#### Solution
+
```

Comment:
I really need a brief explanation of your strategy here in a few sentences. Is it BFS/DFS? What are your terminating conditions? What conditions are you grouping nodes by?

The idea here is that detailing your strategy will help readers understand why you wrote the code that you did, and how it solves the problem you're trying to solve. It would ideally explain the approach the way **you** would explain it if you were facilitating the session on Friday.